srcpkgs/sauerbraten/patches/flags.patch

```diff
@@ -1,35 +1,20 @@
---- a/src/Makefile	2017-01-12 21:54:08.156276696 -0600
-+++ b/src/Makefile	2017-01-12 21:56:06.889752748 -0600
-@@ -1,4 +1,4 @@
--CXXFLAGS= -O3 -fomit-frame-pointer
-+CXXFLAGS= -O3 -fomit-frame-pointer $(CFLAGS)
+--- a/src/Makefile 	2021-12-31 19:09:40.620827928 +0100
++++ b/src/Makefile	2021-12-31 19:11:40.736780250 +0100
+@@ -1,2 +1,2 @@
+-CXXFLAGS= -O3 -fomit-frame-pointer -ffast-math
++CXXFLAGS= -O3 -fomit-frame-pointer -ffast-math $(CFLAGS)
```

Comment:
I am not familiar with this package, but why are we adding -ffast-math here?

Reply:
`fast-math` is decided by upstream, we are only appending `$(CFLAGS)`